msgpack-jackson/src/main/java/org/msgpack/jackson/dataformat/MessagePackFactory.java

```diff
@@ -99,6 +101,22 @@ public JsonParser createParser(byte[] data)
     }
 
     @Override
+    public JsonParser createParser(byte[] data, int offset, int length)
+            throws IOException, JsonParseException
+    {
+        IOContext ioContext = _createContext(data, false);
+        return _createParser(data, offset, length, ioContext);
+    }
+
+    public JsonParser createParser(long memoryAddress, int offset, int length)
```

Comment:
As you might notice since this method doesn't have `@Override` annotation, Jackson doesn't support this kind of API, I think.

How do you use this method via Jackson's API? It would be great, if you add some unit tests to make it clear 😺

Reply:
I use the MessagePackFactory directly, currently using reflection.